modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java

```diff
@@ -47,6 +49,114 @@
  */
 public abstract class BaseExportImportTestCase {
 
+	public void testExportImport(
+			JSONObject actualJSONObject, JSONObject expectedJSONObject,
+			String externalReferenceCode, String name)
+		throws Exception {
+
+		// MVCActionCommand
+
+		MVCActionCommand mvcActionCommand = getMVCActionCommand();
+
+		MockMultipartHttpServletRequest mockMultipartHttpServletRequest =
+			new MockMultipartHttpServletRequest();
+
+		byte[] bytes = JSONUtil.toString(
+			actualJSONObject
+		).getBytes();
```

Comment:
If we change the actualJsonObject to already be a string, we can avoid chaining here